qiskit/providers/ibmq/runtime/runtime_program.py

```diff
@@ -255,3 +258,81 @@ class ProgramResult(NamedTuple):
     name: str
     description: str
     type: str
+
+
+class ParameterNamespace(SimpleNamespace):
+    """ An abstraction for SimpleNamespace that offers param validation.
+
+    Note: Unused/Unnecessary params does not cause error (still validates)
+
+    Args:
+        params (List[ProgramParameter]): The program's input parameters
+
+    Raises
+        IBMQInputValueException on validation fail
+    """
```

Comment:
We usually have separate docstrings for the class itself and the `__init__` method.

Reply:
Fixed!